modules/mono/editor/bindings_generator.cpp

```diff
@@ -939,7 +939,7 @@ void BindingsGenerator::_generate_global_constants(StringBuilder &p_output) {
 			p_output.append(iconstant.proxy_name);
 			p_output.append(" = ");
 			p_output.append(itos(iconstant.value));
-			p_output.append(F != ienum.constants.back() ? ",\n" : "\n");
+			p_output.append(&F != &(ienum.constants.back()->get()) ? ",\n" : "\n");
```

Comment:
Makes sense to me, but I would like @akien-mga to confirm if this is the best approach.

Reply:
This is so confusing because of the variable name. If you see above, we still have the line `const ConstantInterface &iconstant = F;` which is redundant. `F` should be renamed `iconstant`.
Additionally we probably want to cache `&(ienum.constants.back()->get())` into a local outside the loop.

Reply:
I moved all similar cases to outside the loop scope and renamed all the `E` and `F` variables to something more appropriate, removing the redundant variables.